db-commons/src/main/scala/org/bitcoins/db/SlickUtil.scala

```diff
@@ -0,0 +1,21 @@
+package org.bitcoins.db
+
+import scala.concurrent.Future
+import slick.jdbc.SQLiteProfile.api._
+
+sealed abstract class SlickUtil {
+
+  /** Creates rows in a database that are not auto incremented */
+  def createAllNoAutoInc[T](
```

Comment:
@nkohen I couldn't get this to actually work inside of `BlockHeaderDAO`. Basically it comes down to a covariance problem. Could you take a look when you get a chance? I would like to provide a "default" implementation for `createAll`. We now have `CRUDAutoInc` that provides a default implementation for `createAll`, and it would be nice to have a nice helper method for a default implementation of `CRUD.createAll`